charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -529,6 +529,12 @@ admissionWebhooks:
   ## Secret labels
   secretLabels: { }
 
+## Agent workloads list
+agentWorkloads:
+  - name: cloudwatch-agent-with-container-insights-and-application-signals
+    ## Supported modes are daemonset, statefulset and deployment
```

Comment:
any reason to not just expose config here? I know there is a default config, but why not just put the default config directly in the `agentWorkloads` section to make it explicit?